package.json

```diff
@@ -55,6 +56,8 @@
     "@lavamoat/allow-scripts": "^2.3.1",
     "@lavamoat/preinstall-always-fail": "^2.0.0",
     "@metamask/auto-changelog": "^3.4.3",
+    "@metamask/ethjs-contract": "^0.4.1",
+    "@metamask/ethjs-query": "^0.7.1",
```

Comment:
I see that these are kept in  `devDependencies` as tests still rely on them. Is there any concern about using `@ethersproject` packages for tests as well?

Reply:
No concerns only tests needs to be updated with ethersprejects (which is still compatible with them). I am currently doing this